addons/addon-base-raas/packages/base-raas-services/lib/environment/service-catalog/__tests__/environment-sc-connection-service.test.js

```diff
@@ -635,4 +635,49 @@ jM0re//6SUWx/9VfBLN+6Ul8wcqGR2uCmK/PJpzWYxz0IzhnyA==
       );
     });
   });
+
+  describe('verifyAccess', () => {
```

Comment:
Could you also add a test case where admin owns the workspace and the `restrictAdminWorkspaceConnection` flag is set to true?

Reply:
Added as you suggested.